ecs_deploy/cli.py

```diff
@@ -437,13 +441,18 @@ def deploy_task_definition(deployment, task_definition, title, success_message,
                            failure_message, timeout, deregister,
                            previous_task_definition, ignore_warnings, sleep_time):
     click.secho('Updating service')
-    deployment.deploy(task_definition)
+    deploy_response = deployment.deploy(task_definition)
 
     message = 'Successfully changed task definition to: %s:%s\n' % (
         task_definition.family,
         task_definition.revision
     )
 
+    if type(deployment) == DeployBlueGreenAction:
+        _cd_deploy_url = 'https://us-east-1.console.aws.amazon.com/codesuite/codedeploy/deployments/'
```

Comment:
@joaoricardo000 Will it always be `us-east-1`? Or should it be the same region as you are deploying to?

Reply:
I agree, this needs to be configurable, like the regions are in the `deploy`, `scale`, etc. commands.
In addition this should be configured and falling back to a default inside the Action, not in the CLI controller, please.